test/integration/multitenant/multitenant_test.go

```diff
@@ -122,7 +121,8 @@ func TestMultitenant(t *testing.T) {
 						nics := node.Get("networkInterfaces")
 						// for each network interface, verify if it using an external natIP
 						nics.ForEach((func(key, value gjson.Result) bool {
-							assert.Equal(nil, net.ParseIP(value.Get("accessConfigs.0.natIP").String()), "The nodes inside the nodepool should not have external ip addresses.")
+							natIP := value.Get("accessConfigs.0.natIP").String()
+							assert.True((natIP == ""), "The nodes inside the nodepool should not have external ip addresses.")
```

Comment:
Hi @caetano-colin - This should works, but might be more elegant to use `assert.Empty(value.Get("accessConfigs.0.natIP").String())` or `assert.False(value.Get("accessConfigs.0.natIP").Exists())`?

Reply:
thanks for the suggestion -  pushed it on commit 67f6e22